packages/dynamoose/lib/Model/index.ts

```diff
@@ -114,7 +115,6 @@ interface ModelInternalProperties {
 	getHashKey: () => string;
 	getRangeKey: () => string | void;
 	table: () => Table;
-	tableName: string;
```

Comment:
Confused about the purpose behind this change.

Reply:
Just moved tableName property of model from ModelInternalProperties to Model properties

Reply:
@famence Ok, but why?